python/paddle/device/cuda/__init__.py

```diff
@@ -19,6 +24,17 @@
 
 from .streams import Event, Stream
 
+if TYPE_CHECKING:
+    from paddle import CUDAPlace
+    from paddle.base.libpaddle._gpuDeviceProperties import (
+        _gpuDeviceProperties,  # type: ignore
```

Comment:
```suggestion
        _gpuDeviceProperties,  
```

`paddle.base.libpaddle.*`  里面的东西还要看一下该怎么处理 ～ 🤔

参考 https://github.com/PaddlePaddle/Paddle/pull/66051

这里先这么标吧，后面如果确定了再改一下 ～

Reply:
收到